pytorch_lightning/core/lightning.py

```diff
@@ -1220,10 +1220,13 @@ def load_hparams_from_tags_csv(tags_csv):
         logging.warning(f'Missing Tags: {tags_csv}.')
         return Namespace()
 
-    tags_df = pd.read_csv(tags_csv)
-    dic = tags_df.to_dict(orient='records')
-    ns_dict = {row['key']: convert(row['value']) for row in dic}
-    ns = Namespace(**ns_dict)
+    tags = {}
+    with open('meta_tags.csv') as f:
+        csv_reader = csv.reader(f, delimiter=',')
+        for i, row in enumerate(csv_reader):
+            if i != 0:  # header
+                tags[convert(row[0])] = convert(row[1])
```

Comment:
btw, what the convert does?

Reply:
Predefined function in file. It takes in a value that in this case will always be a string (since it comes from the tag file) and convert that to appropriate type meaning:
'true' -> True, '1' -> 1, '2.3' ->2.3 and so on.
Therefore, the line should probably just be:
`tags[row[0]] = convert(row[1])`